### PR TITLE
Implement Cryptographic hash functions

### PR DESCRIPTION
**Pull request by Gokul-Radhakrishnan**
### Description
Used spark's builtin Cryptographic hash functions in PPL

### Issues Resolved
[#759](https://github.com/opensearch-project/opensearch-spark/issues/759)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by YANG-DB — COMMENTED** [review]
Thanks @Gokul-Radhakrishnan for your contribution
Please add an example in the [example commands doc](https://github.com/opensearch-project/opensearch-spark/blob/main/docs/ppl-lang/PPL-Example-Commands.md)
and also add an integration test for an actual values queries from a table in the [ PPL integration test folder](https://github.com/opensearch-project/opensearch-spark/tree/main/integ-test/src/integration/scala/org/opensearch/flint/spark/ppl)
thanks !!

**Comment by YANG-DB** [comment]
@LantaoJin @penghuo please review and add u'r feedback

**Comment by LantaoJin** [comment]
Please add some IntegTests for new functions. It could help us to verify the correctness.

**Comment by LantaoJin** [comment]
https://github.com/opensearch-project/opensearch-spark/blob/main/docs/ppl-lang/README.md needed to update too

**Comment by Gokul-Radhakrishnan** [comment]
Will add the integration tests and update the readme file

**Comment by Gokul-Radhakrishnan** [comment]
> Please add some IntegTests for new functions. It could help us to verify the correctness.

Added Integration tests

**Comment by YANG-DB** [comment]
@Gokul-Radhakrishnan please run `sbt scalafmtAll` prior to commit 
Thanks

**Comment by Gokul-Radhakrishnan** [comment]
@YANG-DB  Integration tests were failing in the build so I fixed. Can you please approve the workflows?

**Review by YANG-DB — APPROVED** [review]

**Comment by YANG-DB** [comment]
@LantaoJin plz review

**Review by LantaoJin — APPROVED** [review]
@Gokul-Radhakrishnan thanks for your contribution. Merging.

File: ppl-spark-integration/src/main/java/org/opensearch/sql/ppl/utils/BuiltinFunctionTranslator.java [hunk 1]
```diff
@@ -13,29 +13,7 @@
...
-import static org.opensearch.sql.expression.function.BuiltinFunctionName.LENGTH;
-import static org.opensearch.sql.expression.function.BuiltinFunctionName.LOCALTIME;
-import static org.opensearch.sql.expression.function.BuiltinFunctionName.MINUTE_OF_HOUR;
-import static org.opensearch.sql.expression.function.BuiltinFunctionName.MONTH_OF_YEAR;
-import static org.opensearch.sql.expression.function.BuiltinFunctionName.SECOND_OF_MINUTE;
-import static org.opensearch.sql.expression.function.BuiltinFunctionName.SUBDATE;
-import static org.opensearch.sql.expression.function.BuiltinFunctionName.TRIM;
-import static org.opensearch.sql.expression.function.BuiltinFunctionName.WEEK;
-import static org.opensearch.sql.expression.function.BuiltinFunctionName.WEEK_OF_YEAR;
+import static org.opensearch.sql.expression.function.BuiltinFunctionName.*;
```
[the remark this answers is not in the document]
Comment (Gokul-Radhakrishnan) [reply ppl-spark-integration/src/main/java/org/opensearch/sql/ppl/utils/BuiltinFunctionTranslator.java:16]: Sure

**Pull request merged**